backend/src/v1/feature_agency_xref/feature_agency_xref.controller.ts

```diff
@@ -0,0 +1,53 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete, UseGuards } from "@nestjs/common";
+import { FeatureAgencyXrefService } from "./feature_agency_xref.service";
+import { CreateFeatureAgencyXrefDto } from "./dto/create-feature_agency_xref.dto";
+import { UpdateFeatureAgencyXrefDto } from "./dto/update-feature_agency_xref.dto";
+import { ApiTags } from "@nestjs/swagger";
+import { UUID } from "crypto";
+import { Roles } from "../../auth/decorators/roles.decorator";
+import { Role } from "../../enum/role.enum";
+import { JwtRoleGuard } from "../../auth/jwtrole.guard";
+
+@UseGuards(JwtRoleGuard)
+@ApiTags("feature-agency-xref")
+@Controller({
+  path: "feature-agency-xref",
+  version: "1",
+})
+export class FeatureAgencyXrefController {
```

Comment:
I would recommend renaming this controller to just `FeatureController` instead of `FeatureAgencyXrefController`

Reply:
@marqueone-ps I was thinking about that but with this current naming, it's easier to know which table it's referring to right of the bat, and the name is consistent with other xref tables like `person-complaint-xref` , `attractant-hwcr-xref` 🤔

Reply:
the `person-complaint-xref` is an exception to some rules, the controller and service logic should actually be in the person controller and service, and the `attractant-hwcr-xref` controller shouldn't exist as the code-table controller actually is what serves up the attractants. At some point there's a lot of clean up we need to do and I've got a feeling we'll probably tackle that when we make the move to emerald.